### ISSUE TITLE
GluonCV 0.3 release tracker

### ISSUE THREAD

**Issue by zhreshold** [dmlc/gluon-cv#271]
This thread is for 0.3 release preparation. 

- [x] Moved tons of image classification models into gluoncv for maintenance. #259, #261
- [x] Faster RCNN on COCO dataset #234 
- [x] YOLOv3 training/val/test #224
- [x] YOLOv3 tutorial #242 
- [x] Mask RCNN training, test #252
- [ ] Mask RCNN tutorial #263
- [x] Custom dataset preparation for object detection #264
- [ ] (optional) C++ deployment tutorial for pre-trained models
- [ ] Fine-tune tutorial for object detection #233, #249 
- [ ] Syncbn for semantic segmentation training

@hetong007 @zhanghang1989 Feel free to add more to the list
